asc2log.c

```diff
@@ -182,6 +183,13 @@ void eval_can(char* buf, struct timeval *date_tvp, char timestamps, char base, i
 		if (strlen(dir) != 2) /* "Rx" or "Tx" */
 			return;
 
+		/* check for signed integer overflow */
+		if (dplace == 4 && read_tv.tv_usec >= INT_MAX/100)
```

Comment:
please add spaces areound the `/`

Reply:
Does it make sense to move the overflow check into `calc_tv()`?

Reply:
added spaces. If check is moved to calc_tv then the struct musst be set to a special value and checked later if time struct is valid, or? Seems to be easier to return before calulating invalid tv.